frontend/packages/topology/src/components/quick-search/QuickSearch.tsx

```diff
@@ -66,7 +66,7 @@ const QuickSearch: React.FC<QuickSearchProps> = ({
   setIsOpen,
 }) => {
   return (
-    <CatalogServiceProvider namespace={namespace}>
+    <CatalogServiceProvider namespace={namespace} catalogId="dev-catalog">
```

Comment:
This should probably use a different catalogId because the items in this catalog might be different than the dev catalog.